grpc-sys/build.rs

```diff
@@ -234,7 +237,78 @@ fn get_env(name: &str) -> Option<String> {
     }
 }
 
+/**
+ * Generate the bindings to C-core
+ */
+fn bindgen_grpc() {
+    let mut config = config_from_headers();
+
+    if get_env("CARGO_CFG_TARGET_OS").map_or(false, |s| s == "windows") {
+        // At lease win7
+        config = config.clang_arg("-D _WIN32_WINNT=0x0700");
+    }
+
+    let bindings = config
+        .clang_arg("-I./grpc/include")
+        .clang_arg("-DGRPC_SYS_SECURE")
```

Comment:
Why define it? What if secure feature is disabled?

Reply:
see [here](https://github.com/pingcap/grpc-rs/blob/master/grpc-sys/grpc_wrap.cc#L782) if the feature is disabled, LLVM can't compile those code > <

Reply:
It's intended not to compile those code when secure feature is disabled. See also https://github.com/pingcap/grpc-rs/blob/d00a8370183a4f0057378cf10cae8476c2dffe64/grpc-sys/build.rs#L303

Reply:
solved